pkg/web/http.go

```diff
@@ -57,8 +62,23 @@ func RunWebServer(config Configuration) error {
 			http.Error(writer, "Error reading request", http.StatusInternalServerError)
 		} else {
 			request := string(body)
-			logrus.Debugf("Request is %v", request)
-			config.Apply(request)
+			//TODO temporary fixes to yaml, which should not be a problem to begin with:
```

Comment:
@swati-kale @ricardozanini The yaml that is generated by the UI has two problems.
1- It has double quotes around it in the actual string, e.g., \"apiVersion: ....\"
2- The newline characters are escaped. Instead of line1\nline2\nline3, it ends up being line1\\nline2\\nline3

The below lines are a hack temporarily to let me move forward, but we need to figure this out.